public/app/features/org/UserInviteForm.tsx

```diff
@@ -60,63 +59,63 @@ const defaultValues: FormModel = {
 
 export const UserInviteForm = () => {
   const dispatch = useDispatch();
+  const {
+    handleSubmit,
+    register,
+    control,
+    formState: { errors },
+  } = useForm<FormModel>({ defaultValues });
 
   const onSubmit = async (formData: FormModel) => {
     await dispatch(addInvitee(formData)).unwrap();
     locationService.push('/admin/users/');
   };
 
   return (
-    <Form defaultValues={defaultValues} onSubmit={onSubmit}>
-      {({ register, control, errors }) => {
-        return (
-          <>
-            <FieldSet>
-              <Field
-                invalid={!!errors.loginOrEmail}
-                error={!!errors.loginOrEmail ? 'Email or username is required' : undefined}
-                label="Email or username"
-              >
-                <Input {...register('loginOrEmail', { required: true })} placeholder="email@example.com" />
-              </Field>
-              <Field invalid={!!errors.name} label="Name">
-                <Input {...register('name')} placeholder="(optional)" />
-              </Field>
-              <Field
-                invalid={!!errors.role}
-                label={
-                  <Label>
-                    <Stack gap={0.5}>
-                      <span>Role</span>
-                      {tooltipMessage && (
-                        <Tooltip placement="right-end" interactive={true} content={tooltipMessage}>
-                          <Icon name="info-circle" size="xs" />
-                        </Tooltip>
-                      )}
-                    </Stack>
-                  </Label>
-                }
-              >
-                <InputControl
-                  render={({ field: { ref, ...field } }) => <RadioButtonGroup {...field} options={roles} />}
-                  control={control}
-                  name="role"
-                />
-              </Field>
-              <Field label="Send invite email">
-                <Switch id="send-email-switch" {...register('sendEmail')} />
-              </Field>
-            </FieldSet>
-            <Stack>
-              <Button type="submit">Submit</Button>
-              <LinkButton href={locationUtil.assureBaseUrl(getConfig().appSubUrl + '/admin/users')} variant="secondary">
-                Back
-              </LinkButton>
-            </Stack>
-          </>
-        );
-      }}
-    </Form>
+    <form onSubmit={handleSubmit(onSubmit)} style={{ maxWidth: 600 }}>
```

Comment:
Been thinking about this `style={{ maxWidth: 600 }}` a bit recently.

I think we need some kind of `<Container />` component to handle this.

Reply:
Something like `<FormContainer/>` or `<NarrowContainer />` or something like that. idk. 

Most pages don't need it, only for forms.

Reply:
Yeah, it's quite annoying, but I don't think that having a separate container just for setting form width is a good idea either. I moved a slightly modified version of the Form to core, so it can be used instead. Ideally we'd need a separate Form component that handles the UI-side only.